### PR TITLE
feat(words): add proxima control words

### PR DESCRIPTION
**Pull request by kosuke55**
add words of https://github.com/tier4/autoware.universe.proxima/pull/10#discussion_r1537537065 cc @masayukiaino

### CODE REVIEW

**Review by Owen-Liuyuxuan — APPROVED** [review]

**Comment by kosuke55** [comment]
added by https://github.com/tier4/autoware-spell-check-dict/pull/692

**Comment by HansRobo** [comment]
Could you resolve conflict?

File: .cspell.json [hunk 1]
```diff
@@ -1326,6 +1346,7 @@
     "PPAGE",
     "pprev",
     "pprint",
+    "PREC",
```
Comment (HansRobo) [inline .cspell.json:1349]: not used in https://github.com/autowarefoundation/autoware.universe/pull/6805?

Comment (HansRobo) [reply .cspell.json:1349]:
```suggestion
```

**Comment by kosuke55** [comment]
@HansRobo thanks!!
@masayukiaino If you will not use this PR, I'll close

**Comment by HansRobo** [comment]
@kosuke55 What's the status of this PR?

**Comment by kosuke55** [comment]
thanks for the remind. 
smart mpc have been merged so I close this PR cc @masayukiaino

**Pull request closed without merging**